lldp_dcbx.c

```diff
@@ -734,12 +734,16 @@ int dcbx_rchange(struct port *port, struct lldp_agent *agent, struct unpacked_tl
 		if (tlv->info[DCB_OUI_LEN] == DCBX_SUBTYPE2) {
 			if (dcbx->dcbx_st == DCBX_SUBTYPE2)
 				dcbx->manifest->dcbx2 = tlv;
+			else
```

Comment:
In this case, perhaps we should introduce a TLV_IGN that we can return?  Then the rxProcessFrame loop will continue processing on the pipeline and if nothing handles it (which we expect in this case) it will free it properly.  WDYT?

Reply:
It looks like elsewhere in this function and other parts of the codebase, they use `SUBTYPE_INVALID` as an equivalent of TLV_IGN.

I could use `SUBTYPE_INVALID` here too, but in this case it is not completely ignored - it sets some flags - and the comment above seems to indicate that it intentional to capture them. I don't know why though?

```
		/* Only store a legacy (CIN or CEE) DCBX TLV which matches
		 * the currently configured legacy dcbx mode.
		 * However, capture if any legacy DCBX TLVs are recieved.
		*/
```